src/localization.js

```diff
@@ -0,0 +1,52 @@
+/*
+Copyright 2020 Google LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+    You may obtain a copy of the License at
+
+https://www.apache.org/licenses/LICENSE-2.0
+
+    Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+    See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+const sourceMap = require('source-map');
+const fs = require('fs');
+
+/**
+ *
```

Comment:
I was more interested in a quick description of what the function does, a.k.a "transform the path of a js file in the web app into its path on disk" or something similar